docker-compose.yml

```
@@ -24,10 +24,15 @@ services:
       - ./dev/vault:/etc/vault
 
   db:
-    image: postgres:10.1
+    image: postgres:10.18
```

```suggestion
    image: postgres:10
```
While we're at it :)